modules/core/pom.xml

```diff
@@ -141,6 +132,34 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <phase>package</phase>
+            <configuration>
+              <relocations>
+                <relocation>
+                  <pattern>org.apache.thrift</pattern>
+                  <shadedPattern>org.shaded.thrfit</shadedPattern>
+                </relocation>
+              </relocations>
+              <artifactSet>
+                <includes>
+                  <include>org.apache.fluo.core.*</include>
+                  <include>org.shaded.thrift:*</include>
+                  <include>org.apache.thrift:*</include>
```

Comment:
I'm not sure these includes are correct. The first one looks like a package pattern. The other two look like maven artifacts, but only the groupId. But, there's no such thing as a groupId named `org.shaded.thrift`.

Reply:
I changed them up. The `org.apache.fluo.core.*` include is needed so the plugin doesn't pull from the entire project but rather just this module. 

The `<include>org.shaded.thrift:*</include>` was needed to include the new shaded artifact and the `<include>org.apache.thrift:*</include>` was needed for the plugin to relocate.